code/modules/mob/living/simple_animal/abnormality/waw/pygmalion.dm

```diff
@@ -128,6 +128,10 @@
 	remove_status_effect(STATUS_EFFECT_SCULPTOR)
 	threat_level = WAW_LEVEL
 	sculptor = null
+	if (sculptor)
+		sculptor.remove_status_effect(STATUS_EFFECT_SCULPTOR)
+	if (missing_prudence)
+		restorePrudence()
```

Comment:
So uh, none of this will work because sculptor is set to "null" before this occurs